modules/ui/src/main/scala/lucuma/ui/primereact/EnumDropdownOptionalView.scala

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.primereact
+
+import cats.syntax.all.*
+import crystal.react.View
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.*
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import react.common.*
+import react.primereact.Dropdown
+import react.primereact.SelectItem
+
+import scalajs.js
+import scalajs.js.JSConverters.*
+
+final case class EnumDropdownOptionalView[A](
+  id:              NonEmptyString,
+  value:           View[Option[A]],
+  exclude:         Set[A] = Set.empty[A],
+  className:       js.UndefOr[String] = js.undefined,
+  clazz:           js.UndefOr[Css] = js.undefined,
+  showClear:       Boolean = true,
```

Comment:
Can we defer the default value to `Dropdown`'s?

Reply:
How could we do that? I don't follow?

Reply:
I meant having `showClear: js.UndefOr[Boolean] = js.undefined` here and let the lower layer decide on the default value, instead of redefining it here.

Reply:
Sure.

Reply:
Wait. Not in this case. The default in `Dropdown` is `false`, but for `View[Option[A]]` the default behavior is to allow clearing. I didn't even include `showClear` at first, but remembered we once had a need for not allowing clearing in an early stage of the proposals ui. I will add a comment to the code.

Reply:
Makes sense. Thank you for the comment